x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/overview_errors/overview_errors.tsx

```diff
@@ -5,35 +5,51 @@
  * 2.0.
  */
 
-import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSpacer, EuiTitle } from '@elastic/eui';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingContent,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
 import React from 'react';
 import { useSelector } from 'react-redux';
 import { i18n } from '@kbn/i18n';
 import { OverviewErrorsSparklines } from './overview_errors_sparklines';
+import { useAbsoluteDate } from '../../../../../hooks';
 import { OverviewErrorsCount } from './overview_errors_count';
 import { ErrorsLink } from '../../../../common/links/view_errors';
 import { selectOverviewStatus } from '../../../../../state';
 
 export function OverviewErrors() {
   const { status } = useSelector(selectOverviewStatus);
 
+  const loading = !status?.enabledIds || status?.enabledIds.length === 0;
```

Comment:
Wouldn't this infinitely load if the user had 0 monitors enabled? in the future for example, if we disabled an entire suite during a maintenance period, or just if they had disabled everything manually for some reason?